Brewfile.lock.json

```diff
@@ -95,45 +95,84 @@
         }
       },
       "swiftformat": {
-        "version": "0.52.11",
+        "version": "0.53.0",
```

Comment:
Are there any places in the project where the formatting should have changed due to the formatter update?

Reply:
Oh yes actually 4 total; this adds one newline and changes 3 `.forEach` instances into `for` loops